packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -148,18 +163,28 @@ class AnalyzeCommand extends FlutterCommand {
 
     if (argResults['current-package']) {
       // ./lib/main.dart
-      String mainPath = 'lib/main.dart';
+      String mainPath = path.join('lib', 'main.dart');
```

Comment:
I'm used to using `join` for cross-platformness but it may not be necessary. I'll revert this bit.

In other packages, I've had convenience methods to convert unix paths to platform paths. So a constant `lib/main.dart` converted by replaceAll('/', Platform. pathSeparator).